### DISCUSSION TITLE
节标题缩进的中英文差异建议

### DISCUSSION THREAD

**Discussion by nyLiao** [sjtug/SJTUThesis#604] [提问] [answered]
本科生论文 Word 模版中关于二三四级标题缩进，中英文模版要求是不同的：

* 中文：

> 二级标题为黑体四号，单倍行距，*首行缩进2字符*。

> 第三级和第四级标题小四宋体书写标题，*缩进值2字符*。

* 英文：

> 二、三、四级标题均为Times New Roman，14号字加粗*左对齐*，上下各空一行。

目前按 `indent-headings` 默认设置，会导致英文文档标题也缩进 2 字符。可能需要添加按语言调整 `indent-headings` 的判断？

**Comment by PaulLiuYZ** [comment]
想问你是怎么解决这个问题的呢

**Comment by nyLiao** [comment, marked as answer]
> 想问你是怎么解决这个问题的呢

目前的话是在 `sjtusetup.tex` 文件里加上 `style = {indent-headings = false}`。因为没搞清楚硕士博士类型的要求，所以也没好修改模版提 PR。
